crypto/keys/secp256k1/secp256k1.go

```diff
@@ -21,16 +21,63 @@ import (
 
 var (
 	_ cryptotypes.PrivKey  = &PrivKey{}
+	_ cryptotypes.PrivKey  = &PrivKeyOld{}
 	_ codec.AminoMarshaler = &PrivKey{}
+	_ codec.AminoMarshaler = &PrivKeyOld{}
 )
 
 const (
-	PrivKeySize = 32
-	keyType     = "secp256k1"
-	PrivKeyName = "tendermint/PrivKeySecp256k1"
-	PubKeyName  = "tendermint/PubKeySecp256k1"
+	PrivKeySize    = 32
+	keyType        = "secp256k1"
+	PrivKeyNameOld = "tendermint/PrivKeySecp256k1"
+	PubKeyNameOld  = "tendermint/PubKeySecp256k1"
+	PrivKeyName    = "comet/PrivKeySecp256k1Uncompressed"
+	PubKeyName     = "comet/PubKeySecp256k1Uncompressed"
 )
 
+func (privKey *PrivKeyOld) Bytes() []byte {
+	return privKey.Key
+}
+func (privKey *PrivKeyOld) PubKey() cryptotypes.PubKey {
```

Comment:
Not sure but do the `PrivKeyOld` methods call the respective`PrivKey` implementation ? And same for `PubKeyOld` ?

Reply:
That's a very nice and subtle catch, thanks!
Porting was almost 1:1. but here we need to copy-paste the function.

Reply:
Addressed [here](https://github.com/0xPolygon/cosmos-sdk/pull/8/commits/d3b8a83f0346a965c1610a076c7f31081862cbde)

Reply:
I think you'd have to extend it to all `PrivKeyOld` methods 😅

Reply:
Yeah some needed to be changed, thanks.
Addressed [here](fdf810ea8b6823862e3126b01313f3707dcda6e2)